### ISSUE TITLE
Setting objects before starting the map itself (taiko editor)

### ISSUE THREAD

**Issue by Moltanicaa** [ppy/osu#9733]
**Describe the bug:** 
I can place objects in front of the beginning of the map, and visually the place of installation is distorted. I will not be able to place them much earlier than the map itself begins. Because of this, it is impossible to place objects earlier, even if the soundtrack itself is going as usual.
**Screenshots or videos showing encountered issue:** 
![image](https://user-images.githubusercontent.com/59089206/89032197-7b371d80-d377-11ea-97ef-0800d2629247.png)
**osu!lazer version:** 
2020.725.0

**Comment by bdach** [comment]
Related to #9628. A general direction whether it is allowed to place object before time 0 should be enforced across the board.

**Comment by peppy** [comment]
Closing as duplicate of #9628. This doesn't need to be tracked on its own.
